### PR TITLE
Add simple-chatbot-server in tools.

### PR DESCRIPTION
**Pull request by liayoo**
- Added simple-chatbot-server named `echo-bot` to tools

### CODE REVIEW

File: tools/simple-chatbot-server/index.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const AinJs = require('@ainblockchain/ain-js').default;
+const CommonUtil = require('../../common/common-util');
+const app = express();
+const port = 80;
+const blockchainEndpoint = 'http://dev-node.ainetwork.ai:8080/';
+const BOT_NAME = 'echo-bot';
+const BOT_PK = 'ee0b1315d446e5318eb6eb4e9d071cd12ef42d2956d546f9acbdc3b75c469640';
```
Comment (platfowner) [inline tools/simple-chatbot-server/index.js:8]: minor: what about moving it to line 10 (before 'const BOT_ADDRESS ...')?

File: tools/simple-chatbot-server/index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  const ref = tx.tx_body.operation.ref;
+  const parsedRef = CommonUtil.parsePath(ref);
+  const userVal = tx.tx_body.operation.value;
+  if (parsedRef.length !== 6 || parsedRef[0] !== 'apps' || parsedRef[2] !== 'messages' ||
+      parsedRef[5] !== 'user') {
+    console.log(`Not supported path pattern: ${ref}`);
+    return;
+  }
+  const answerRef = CommonUtil.formatPath([...parsedRef.slice(0, parsedRef.length - 1), BOT_NAME]);
+  const result = await ain.db.ref(answerRef).setValue({
```
Comment (platfowner) [inline tools/simple-chatbot-server/index.js:44]: cool~

**Review by platfowner — APPROVED** [review]
LGTM with some minor comments.
Thanks!

File: tools/simple-chatbot-server/index.js (hunk 1 above)
Comment (liayoo) [reply tools/simple-chatbot-server/index.js:9]: Done!

File: tools/simple-chatbot-server/index.js (hunk 2 above)
Comment (liayoo) [reply tools/simple-chatbot-server/index.js:44]: 😎

**Comment by liayoo** [comment]
Thanks for the quick review! Echo bot is now alive! 🤖

**Pull request merged**